packages/aws-cdk-lib/aws-codebuild/test/project.test.ts

```diff
@@ -180,6 +180,31 @@ describe('GitHub source', () => {
     });
   });
 
+  test('can create organizational webhook', () => {
+    // GIVEN
+    const stack = new cdk.Stack();
+
+    // WHEN
+    new codebuild.Project(stack, 'Project', {
+      source: codebuild.Source.gitHub({
+        owner: 'testowner',
+      }),
+    });
+
+    // THEN
+    Template.fromStack(stack).hasResourceProperties('AWS::CodeBuild::Project', {
+      Source: {
+        Type: 'GITHUB',
+        Location: 'CODEBUILD_DEFAULT_WEBHOOK_SOURCE_LOCATION',
+      },
+      Triggers: {
+        ScopeConfiguration: {
+          Name: 'testowner',
+        },
+      },
+    });
+  });
```

Comment:
can we also test `webhookFilter` here since webhook filter's logic also changes if it's organization github source.

Reply:
Yeah I'll add tests for FilterGroups.

Reply:
Done!